test/browser_chrome/browser_experiment_installed_check.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// (Modeled after Screenshots system add-on tests in browser/extensions/screenshots/test)
+
+/* global ChromeUtils, AddonManager, add_task, ok, is */
```

Comment:
Should probably add an eslint config so we don't need to do this... normally we'd inherit the `browser/` or `toolkit/` one but I guess there isn't much JS in `testing/` :)

Reply:
Note that it's not important now, but will be a pain as more tests are added.